mspray/apps/main/tests/views/test_user.py

```diff
@@ -0,0 +1,66 @@
+"""Test user login and logout module."""
+from unittest.mock import patch
+from django.test import TestCase
+
+from mspray.apps.main.views.user import get_form_users
+
+form_data = {
+  "url": "https://api.ona.io/api/v1/forms/344713.json",
+  "formid": 344713,
+  "users":
+  [{"first_name": "Derek", "last_name": "Pollard",
+  "user": "dpollard", "role": "owner",
+      "is_org": False},
+      {
+      "first_name": "James Mulwanda",
+      "last_name": "Simfukwe",
+      "user": "james2207",
+      "role": "owner",
+      "is_org": False},
+      {
+      "first_name": "Akros",
+      "last_name": "Mspray",
+      "user": "msprayzambia2017",
+      "role": "dataentry",
+      "is_org": False},
+      {
+      "first_name": "Anne",
+      "last_name": "Martin",
+      "user": "acmartin",
+      "role": "owner",
+      "is_org": False},
+      {
+      "first_name": "Craig",
+      "last_name": "",
+      "user": "cappl",
+      "role": "readonly",
+      "is_org": False},
+      {
+      "first_name": "Ukang'a",
+      "last_name": "Dickson",
+      "user": "ukanga",
+      "role": "owner",
+      "is_org": False},
+      {
+      "first_name": "casius",
```

Comment:
I think these are names of actual people that we work with.  We should change them.